vips/vip-converter/vip-converter-testnet/vip-converter2.ts

```diff
@@ -0,0 +1,186 @@
+  incentiveAndAccessibilityForRiskFundConverter,
+  incentiveAndAccessibilityForUSDCPrimeConverter,
+  incentiveAndAccessibilityForUSDTPrimeConverter,
+  incentiveAndAccessibilityForXVSVaultConverter,
+} from "./commands";
+
+const DEFAULT_PROXY_ADMIN = "0x7877fFd62649b6A1557B55D4c20fcBaB17344C91";
+const RISK_FUND_PROXY = "0x487CeF72dacABD7E12e633bb3B63815a386f7012";
+const RISK_FUND_V2_IMPLEMENTATION = "0x217a907B0c6a7Dc67a21F769a915722B98136F82";
+
+const PROTOCOL_SHARE_RESERVE_PROXY = "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3";
+const PROTOCOL_SHARE_RESERVE_NEW_IMPLEMENTATION = "0x194777360f9DFAA147F462349E9bC9002F72b0EE";
+
+const VTREASURY = "0x8b293600C50D6fbdc6Ed4251cc75ECe29880276f";
+const XVS_VAULT_TREASURY = "0xab79995b1154433C9652393B7BF3aeb65C2573Bd";
+
+export const vipConverter2 = () => {
+  const meta = {
+    version: "v2",
+    title:
+      "VIP-converter2 Upgrades the implementation of RiskFund and ProtocolShareReserve with Adding of converts in ConverterNetwork and vice versa. It also sets conversion configs for the converters",
+    description: `
+    Gives call permissions to timelock
+    Upgrade the implementation of riskfund to riskfund V2
+    sets RiskFundConverter in RiskFundV2
+    Upgrade the implementation of ProtocolShareReserve and update the distributionConfigs
+    update destination address to Riskfund in RiskFundConverter
+    Add Converters in ConverterNetwork
+    Add ConverterNetwork in Converters
+    Sets Conversion configs for the following converters:-
+    1. RiskFundConverter
+    2. USDTPrimeConverter
+    3. USDCPrimeConverter
+    4. BTCBPrimeConverter
+    5. ETHPrimeConverter
+    6. XVSVaultConverter`,
+
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      grant(RISK_FUND_CONVERTER, "setPoolsAssetsDirectTransfer(address[],address[][],bool[][])", NORMAL_TIMELOCK),
+      grant(RISK_FUND_CONVERTER, "setPoolsAssetsDirectTransfer(address[],address[][],bool[][])", FAST_TRACK_TIMELOCK),
+      grant(RISK_FUND_CONVERTER, "setPoolsAssetsDirectTransfer(address[],address[][],bool[][])", CRITICAL_TIMELOCK),
+
+      grant(CONVERTER_NETWORK, "addTokenConverter(address)", NORMAL_TIMELOCK),
+      grant(CONVERTER_NETWORK, "addTokenConverter(address)", FAST_TRACK_TIMELOCK),
+      grant(CONVERTER_NETWORK, "addTokenConverter(address)", CRITICAL_TIMELOCK),
+
+      grant(CONVERTER_NETWORK, "removeTokenConverter(address)", NORMAL_TIMELOCK),
+      grant(CONVERTER_NETWORK, "removeTokenConverter(address)", FAST_TRACK_TIMELOCK),
+      grant(CONVERTER_NETWORK, "removeTokenConverter(address)", CRITICAL_TIMELOCK),
+
+      grant(XVS_VAULT_TREASURY, "fundXVSVault(uint256)", NORMAL_TIMELOCK),
+      grant(XVS_VAULT_TREASURY, "fundXVSVault(uint256)", FAST_TRACK_TIMELOCK),
+      grant(XVS_VAULT_TREASURY, "fundXVSVault(uint256)", CRITICAL_TIMELOCK),
```

Comment:
I would move these commands to the first VIP, so we can say:
* VIP 1: ownership a permissions
* VIP 2: contract configurations

Reply:
I also wanted the same but due to large number of commands in VIP 1, i had to shift these commands to VIP2.

Reply:
https://github.com/VenusProtocol/vips/pull/49/commits/9eb73c07ce3c22d0e06dac9f5ba757469d673f1c